2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,14 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  let sutibleAricleTiteles = [];
```

Comment:
Be cautious of spelling mistakes, I believe this should be `suitableArticleTitles` :)

Reply:
Yes I should change it.
Thanks